packages/teams-js/jest.config.cjs

```diff
@@ -1,11 +1,12 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
+/* eslint-disable @typescript-eslint/no-var-requires */
 const commonSettings = require('../../jest.config.common.js');
 const packageVersion = require('./package.json').version;
 
 module.exports = {
   ...commonSettings,
   globals: {
     PACKAGE_VERSION: packageVersion,
+    fetch: global.fetch,
   },
-  //setupFilesAfterEnv: ['./jest-setup.cjs'],
+  setupFilesAfterEnv: ['./test/setupTest.ts'],
```

Comment:
Why is this change needed?

Reply:
This change is needed since we need to execute the setupTest file as setup prior to any test running. That file allows us to create the `fetch` polyfill.